core/lexicon/en/file.inc.php

```diff
@@ -117,7 +118,7 @@
 $_lang['upf_close'] = 'Close';
 $_lang['upf_close_desc'] = 'Close the window.';
 $_lang['upf_err_failed'] = 'Server is unavailable or internal server error occured.';
-$_lang['upf_err_filetype'] = 'Selected file extension isn\'t permitted.<br/>Please select files with following extensions: {1}';
+$_lang['upf_err_filetype'] = 'Selected file extension isn’t permitted.<br>Please select files with following extensions: {1}';
```

Comment:
I am fine using the right typography here, but the whole docs are different with single quotes.

Reply:
It is better to return it as it was, because all changed strings will overwrite translations in Crowdin.
After a few of these iterations, MODX will lose all translations.
p.s. And for the new line, leave it in the old style: firstly, it will look in one style, and secondly, there will be no confusion with quotes.